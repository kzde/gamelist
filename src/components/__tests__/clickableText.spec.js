import React from 'react';
import renderer from 'react-test-renderer';
import ClickableText from '../ClickableText';

describe('ClickableText', () => {
  it('should renders correctly', () => {
    const tree = renderer.create(<ClickableText text="link test" />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
