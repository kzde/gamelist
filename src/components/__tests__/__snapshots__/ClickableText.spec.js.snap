// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ClickableText should renders correctly 1`] = `
<div
  className="ClickableText__Wrapper-sc-1hvvc01-0 gReZzT"
>
  <span
    className="ClickableText__Content-sc-1hvvc01-1 khlrPb"
    data-hover="link test"
  >
    link test
  </span>
</div>
`;
